docs/js/react.js

```diff
@@ -1109,6 +1109,9 @@ var ReactClass = {
       this.props = props;
       this.context = context;
       this.refs = emptyObject;
+      if (this.updater && "development" !== 'production') {
```

Comment:
This is not a source file. We have a transform that converts `if (__DEV__)` to the output you see here.